README.md

```diff
@@ -125,8 +140,66 @@ To update Spdx-Java-Library, the following is a very brief checklist:
 3. If there are any conversions that are needed when copying to or from the new model version, add conversion code to the `ModelCopyConverter` class.
 4. Update SpdxModelFactory unit test for the highest version check
 
-## Development Status
+## API Documentation
```

Comment:
Perhaps this subsection should be higher in the README?  Often reference documentation is accessed repeatedly by the developer, while usage documentation (which is mostly what the rest of the README contains) is only accessed early on in a developer's journey with a library.

Reply:
Agree. Moved up.